### DISCUSSION TITLE
Multiplayer gamemode can be overwritten by selecting map of different gamemode

### DISCUSSION THREAD

**Discussion by YonatonC** [ppy/osu#16349] [Q&A] [answered]
In a multiplayer lobby by adding a beatmap with a gamemode different than the next map to the queue, your gamemode will be a convert of the gamemode you selected instead of the correct gamemode.

Steps to reproduce:
1. Add a map to the top of the queue
2. Add another map, but select a different gamemode
3. Your gamemode will be the one selected in step 2, not step 1

Video example: https://www.youtube.com/watch?v=e0JIPjkrTS4

**Comment by bdach** [comment, marked as answer]
#16351
